client/landing/stepper/declarative-flow/internals/steps-repository/goals/goals.tsx

```diff
@@ -75,12 +75,21 @@ export const useGoals = (): Goal[] => {
 		},
 	];
 
-	const hideDIFMGoalForNonEN = ( { key }: Goal ) => {
```

Comment:
Minor and not important but, this can be simplified

```typescript

	/**
     * Conditionally hide goals
	 * -  Hides the DIFM goal for all locales except English and ES.
	 */
    const difmVisibleLocales = [ ...englishLocales, 'es' ];
	const filterGoals = ( { key }: Goal ) => {
		if ( key === SiteGoal.DIFM || !difmVisibleLocales.includes(locale) ) {
			return false;
		}
		return true;
	};
```

Reply:
Thanks! This does simplify it!